sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/ObjectAnchorsModelFactory.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.MixedReality.ObjectAnchors.Models;
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary>
+    /// Object Understanding model factory that enables mocking for the Object Understanding client library.
+    /// </summary>
+    public static class ObjectAnchorsModelFactory
```

Comment:
I really don't understand this class. Why not just make the constructors of the others public?

Reply:
In response to this: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-mocking